addons/mrp/mrp.py

```diff
@@ -264,6 +264,17 @@ def _skip_bom_line(self, cr, uid, line, product, context=None):
                 return True
         return False
 
+    def _prepare_conssumed_line(self, cr, uid, bom_line_id, quantity, product_uos_qty, context=None):
```

Comment:
can you rename bom_line_id into bom_line ? Indeed it's not a id even if in the original code then call it bom_line_id I think it's a bad idea to keep this name here